Template and VF Components/TestQuery.cls

```diff
@@ -0,0 +1,44 @@
+@isTest
+public class TestQuery 
```

Comment:
Class name is wrong.
Test class name should be Like
`<ClassTobeTested>` Test

Reply:
changed it.